### ISSUE TITLE
New buttons ("Stepping" and "Play/Pause") doesn't seem to working

### ISSUE THREAD

**Issue by adventur-help0690** [adventur-help0690/Astar_vs_Dijkstra#2]
This is a whole big problem
1. There is no last step showing how the algorithm confirm the result path (yellow).
2. Clicking on either button will do nothing when no runs were did. This should be a warn, or more better - hide them when the user didn't click "Run Algorithm"
3. It didn't work for A*. Clicking on "Stepping" or "Play/Pause" does nothing.
And there's more. But I suggest myself to work one by one.
